pages/dkp/kaptain/1.2.0/install/on-premise/index.md

```diff
@@ -41,6 +41,7 @@ The steps to install Kaptain on an on-premises cluster are as follows:
         - name: knative
           enabled: true
     ```
+<!-- Do we need to update anything on the file location above? update/decompress the link?-->
```

Comment:
Same as the previous comment - this is just for the knative install. NO MinIO here.